api/src/main/java/org/openmrs/validator/PersonAddressValidator.java

```diff
@@ -63,21 +63,15 @@ public void validate(Object object, Errors errors) {
 			log.debug(this.getClass().getName() + ".validate...");
 		}
 		
-		/*if (object == null) {
-			throw new IllegalArgumentException("The personAddress object should not be null");
-		}*/
+		if (object == null) {
+			System.out.println("The personAddress object should not be null");
+
+			System.exit(0); // Terminates JVM
```

Comment:
Don't exit the complete program by placing an exit(). This error is not so bad that his should happen. Neither it does in other validatios.

What you did previously was the correct approach. Just avoid executing the part after the catch block. If the following statement in Line 83 gets executed (personAddress.getAddress1()) and personAdress is null a non-catched exception will be thrown. 

Consider one of the following options:
(1)
```
if (personAddress == null) {}
else { //add else part here }	

```
OR 
(2)

```
> if (personAddress == null) {
> ......
> return 
> }	
```

Reply:
+ Consider squashing both commits into one  https://wiki.openmrs.org/display/docs/Git-Squashing+commits